src/mscorlib/src/System/Char.cs

```diff
@@ -203,7 +203,7 @@ public bool Equals(Char obj)
       [Pure]
       public static bool IsDigit(char c) {
           if (IsLatin1(c)) {
-            return (c >= '0' && c <= '9');
+            return (uint)(c - '0') <= (uint)('9' - '0');
```

Comment:
Self-note: I'm considering moving all of these boolean returns into a private helper method like the following:

```cs
private bool IsBetweenInclusive(char lowerBound, char upperBound)
{
    return (uint)(m_value - lowerBound) <= (uint)(upperBound - lowerBound);
}

// Usage:
c.IsBetweenInclusive('0', '9');
```

This way we avoid repeating the value for the lower bound, and is less verbose/error prone.